flintrock/services.py

```diff
@@ -106,11 +115,18 @@ def health_check(
         """
         raise NotImplementedError
 
+    def create_security_groups(self, flintrock_client_cidr: str):
+        """
+        Create and return the EC2 SecurityGroupRules required by this service
```

Comment:
[Nit](https://stackoverflow.com/a/27810540/877069): Full stop at the end of the sentence, please.